### DISCUSSION TITLE
Using modals

### DISCUSSION THREAD

**Discussion by Elmatbaisaad** [symfony/symfony#46983] [Q&A] [answered]
Hello everyone ! 
i want to use modal in one of my pages but there no pop up i don't understand why here is the code on the twig :
![image](https://user-images.githubusercontent.com/101296742/179756434-456881b8-6aab-43b2-bc04-999c577f5b57.png)
Thank you !

**Comment by DylanKas** [comment, marked as answer]
This is not related to symfony.
Do you use bootstrap ? Did you properly install it ?

Your question may be more fitted on stack overflow (with more details and debugging information).
